scenario_player/runner.py

```diff
@@ -152,12 +153,19 @@ def determine_run_number(scenario_dir: Path) -> int:
     return run_number
 
 
-def make_session(auth: str, settings: SettingsConfig) -> Session:
+def make_session(auth: str, settings: SettingsConfig, node_config: NodesConfig) -> Session:
+    num_connections = node_config.count * 10
+
     session = Session()
     if auth:
         session.auth = cast(Tuple[str, str], tuple(auth.split(":")))
-    session.mount("http", TimeOutHTTPAdapter(timeout=settings.timeout))
-    session.mount("https", TimeOutHTTPAdapter(timeout=settings.timeout))
+    session.mount(
+        "http", TimeOutHTTPAdapter(timeout=settings.timeout, pool_connections=num_connections)
```

Comment:
@ulope Any reason why we instantiate two `TimeOutHTTPAdapter`s here? Would one be enough?

Reply:
Not sure tbh. The example in the requests documentation was doing it this way.